jupyterlab/commands.py

```diff
@@ -975,14 +975,20 @@ def unlink_package(self, path):
         self._write_build_config(config)
         return True
 
-    def toggle_extension(self, extension, value):
+    def toggle_extension(self, extension, value, user=False, sys_prefix=False):
```

Comment:
This API presents `user` and `sys_prefix` as independent options, but they are really the same option, right? Is there a reason we're not using the similar `level` parameter, which can mutually exclusively be `'user'`, `'sys_prefix`', or `'system'`? Then we don't have to think about the precedence of the options?